src/components/Dashboard/Dashboard.js

```diff
@@ -19,6 +40,45 @@ export const Dashboard = () => {
           <span className="arrow"></span>
         </div>
       </header>
+      <div className="dpsg-wrapper">
+        <div className="dpsg-content-wrap">
+          <div className="dp-container">
+            <S.Subtitle>
+              <span>
+                <img
+                  src="https://cdn.fromdoppler.com/doppler-ui-library/latest/img/asset-enterprise.svg"
+                  alt="icon-title"
+                />
+              </span>
+              Aprende con Doppler
+            </S.Subtitle>
+            <div className="dp-rowflex">
+              <div className="col-sm-12 col-md-6">
+                <Carousel id={'1'} color={'orange'}>
+                  {({ activeSlide }) =>
+                    postList.map((post, index) => (
+                      <Slide key={post.id} active={activeSlide === index}>
+                        <TextPreviewPost post={post} />
+                      </Slide>
+                    ))
+                  }
```

Comment:
Why not remove the the `{}` surrounding the function parameter?
```suggestion
                  {( activeSlide ) =>
                    postList.map((post, index) => (
                      <Slide key={post.id} active={activeSlide === index}>
                        <TextPreviewPost post={post} />
                      </Slide>
                    ))
                  }
```

Reply:
could be removed. But, if is removed then should be removed from Carousel.js component
`children(activeSlide)` instead `children({ activeSlide })`

Reply:
removing that only one carousel works

Reply:
It appeared to be not neccesary but it looks it does. Thanks for the clarification.